### ISSUE TITLE
gauge does not work with python '3.7.4' version

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/gauge-python#164]
### Excepted Result:
gauge should work seamlessly with python '3.7.4' version 

### Actual Result:
we are getting the below log
'"gauge -v --machine-readable"' is not recognized as an internal or external command,
operable program or batch file.

Add the expected behavior/issue description/Actual behavior here

### Steps to reproduce
- create a gauge-python project `gauge init python`
- do `gauge run`

### Information that will help us debug
Tested version:
```
python version: 3.7.4
Gauge version: 1.0.6
Commit Hash: 2bc49db

Plugins
-------
html-report (4.0.8)
python (0.3.6.nightly-2019-10-14)
commit: c455c7544676876001bf696ca3b8b8ea6b13a166
screenshot (0.0.1)
```

**Comment by Apoorva-GA** [comment]
Fix available in commit : 5c58339ccaea7c3f13456ef9478ac3989f9dd31f

**Comment by Debashis9012** [comment]
This issue has been tested and verified.
commit: 5c58339
